install/kubernetes/cilium/values.yaml

```
@@ -2801,6 +2804,12 @@ clustermesh:
     # -- Number of replicas run for the clustermesh-apiserver deployment.
     replicas: 1
 
+    # -- lifecycle setting for the apiserver container.
+    lifecycle: {}
+
+    # -- terminationGracePeriodSeconds(default 30s) for the clustermesh-apiserver deployment.
```

nit: no need to specify the default, the helm doc table has a column for the value and when reading the value file we can read the actual default value.
```suggestion
    # -- terminationGracePeriodSeconds for the clustermesh-apiserver deployment.
```

---

okay, let me update the PR, thanks.